crown-apparel/install.sh

```diff
@@ -0,0 +1,153 @@
+#!bin/bash
+echo -e "==== Beginning install ====\n"
+
+if ( apt-cache show nodejs ) 
+then
+  echo -e "\n==== Node setup already installed ===="
+else 
+  echo -e "\n==== Checking if Nodejs setup is installed ===="
+  curl -fsSL https://deb.nodesource.com/setup_19.x | sudo -E bash - 
+fi
+
+# make sure node & npm is installed
+if ( which node ) 
+then
+  echo -e "\n==== Node already installed ===="
+else 
+  echo -e "\n==== Installing Nodejs && npm ===="
+  sudo apt install -y nodejs
+fi
+
+# cd into the crown app directory
+if [[ ${PWD} = */crown-apparel/* ]]
```

Comment:
In the deploy you are just using a `cd` to the right directory, do we need to check again here? What is the scope of concern here and is this reaching to far out of it from the install.sh perspective?

Reply:
Yep, thats right. I removed it. Maybe confused ~/crownapp with ~/crown-apparrel